pkg/pool-manager/pool.go

```diff
@@ -176,3 +177,24 @@ func getNextMac(currentMac net.HardwareAddr) net.HardwareAddr {
 
 	return currentMac
 }
+
+// Checks if the namespace of an instance is opted in for kubemacpool
+func (p *PoolManager) IsInstanceOptedIn(namespaceName string, labelSelector string) (error, bool) {
+
+	isNamespaceOptedIn := false
+	ns, err := p.kubeClient.CoreV1().Namespaces().Get(namespaceName, metav1.GetOptions{})
+
```

Comment:
Error handling should follow the line that returned the error. Without a space in between it is clearer who's error we handle.